app/views/bikes/show.html.haml

```diff
@@ -141,7 +141,7 @@
                 = @bike.attr_list_item(@stolen_record.address(skip_default_country: true), t(".location"))
                 = @bike.attr_list_item(@stolen_record.locking_description, t(".locking_description"))
                 = @bike.attr_list_item(@stolen_record.lock_defeat_description, t(".locking_circumvented"))
-                - stolen_type = @bike.recovered ? 'recovered' : 'stolen'
+                - stolen_type = @bike.abandoned ? 'recovered' : 'stolen'
```

Comment:
Update text here. Probably need to change some of the text as well, in the translation file